addons/sale/models/sale_order.py

```diff
@@ -251,6 +251,11 @@ def _rec_names_search(self):
         help="Internal warning for the partner or the products as set by the user.",
         compute='_compute_sale_warning_text')
 
+    display_taxes = fields.Boolean(
+        compute='_compute_display_taxes',
+        help="Show taxes in the report"
+    )
+
```

Comment:
There is no need to create a field if it's not stored or shown in a backend view.
There is already a `display_discount` in the templates, please do the same for taxes.